com.unity.shadergraph/Editor/Drawing/Views/MaterialGraphView.cs

```diff
@@ -1207,7 +1207,7 @@ internal static void InsertCopyPasteGraph(this MaterialGraphView graphView, Copy
                         foreach (var node in dependentPropertyNodes)
                         {
                             node.owner = graphView.graph;
-                            node.property = property;
+                            node.property = (AbstractShaderProperty)copiedInput;
```

Comment:
Nice catch on these, looks like that `copiedInput` has been unused since it was written last March. =/

I'd prefer to do a little cleanup and get rid of the top-level `copiedInput` entirely, just do `var copiedProperty = DuplicateShaderInputs(...) as ASP;` inside this switch (same for keywords below).